provider/kubernetes/percentage.go

```diff
@@ -0,0 +1,44 @@
+package kubernetes
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// PercentageValue is int64 form of percentage value with 10^-3 precision.
+type PercentageValue int64
+
+// PercentageValueFromString tries to read percentage value from string like "1.1%", "6%".
+// It will lose the extra precision if there are more than 3 digit after decimal point.
+func PercentageValueFromString(s string) (PercentageValue, error) {
+	idx := strings.Index(s, "%")
+	if idx < 0 {
+		return 0, fmt.Errorf("missing %% for parsing percentage value: %q", s)
```

Comment:
I don't know if the `%` value is required.....is it really adding anything? You are still using `ParseFloat` on the string anyways....why not just strip it off if it is there, then continue on as usual. If you were parsing percentages differently than raw values, sure, but that doesn't appear to be the case here.